src/components/organisms/Sidebar/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { Stack } from '@chakra-ui/react';
+import Logo from 'components/atoms/Logo';
+import React from 'react';
+import SidebarNav from 'components/organisms/SidebarNav';
+
+const Sidebar = () => (
```

Comment:
You probably said that on our last meeting, but I don't remember precisely.
Why do you define separate nested folder with `index.tsx` file instead of just making `Sidebar.tsx`?
It's a little bit verbose for me.

Reply:
The logic behind this is that one folder corresponds to one component. This way I place three files as standard:

1. `name-component.tsx` - component implementation
2. `component-name.styles.ts` - styles for the component
3. `name-component.spec.ts` - tests
Sometimes I also put a `stories` dedicated folder for the storybook.

`index.ts` just includes component export and possible dependency imports. I use this so that when importing elsewhere in the code I don't have to write:

```
import { ComponentName } from 'components/ComponentName/ComponentName'
```

Instead I can omit filename:
```
import { ComponentName } from 'components/ComponentName'
```
because Node by default looks for index file in any path.

As you can see, I have implementation in index.ts, because I don't have dedicated styles. When the component will "grow" I will separate the logic into `ComponentName.tsx` and styles into `ComponentName.styles.ts`, and `index.ts` will contain only the component export. I have everything grouped in a directory, so the directory structure will not change.

Reply:
Great! Thanks for depth clarification. 👍